app/helpers/account_helper.rb

```diff
@@ -46,7 +46,7 @@ def account_states_for_select
   def account_states_info(account)
     state_info = account.state.humanize
     if account.scheduled_for_deletion?
-      "#{state_info} (#{account.deletion_date.to_date.to_s(:long)})"
+      "#{state_info} (#{account.deletion_date.to_date.to_s})"
```

Comment:
Use I18n

On rails upgrade this will break

https://guides.rubyonrails.org/i18n.html#adding-date-time-formats